packages/sfui/frameworks/react/components/SfIcons/SfIconAddShoppingCart.tsx

```diff
@@ -1,14 +1,11 @@
+
 import type { SfIconProps } from '@storefront-ui/react';
 import { SfIconBase, SfIconSize } from '@storefront-ui/react';
 
 export default function SfIconAddShoppingCart({
-  size = SfIconSize.base,
-  viewBox = '0 0 24 24',
-  ...attributes
+    size = SfIconSize.base,
```

Comment:
question: why there is change in icons?

Reply:
I think those are because you have version branch before update dependencies, please rebase with v2-develop and check again